### PR TITLE
Feedback school search/submission analytics

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes add various analytics events to the feedback tool.
- school not found
- submission successful
- submission failed
- renamed "complaint" to "feedback"

There is an [outstanding question](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/11265#issuecomment-422144856) regarding the missing-information event.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by erikphansen — APPROVED** [review]

File: src/applications/edu-benefits/feedback-tool/helpers.js [hunk 1]
```diff
@@ -125,9 +127,7 @@ export function submit(form, formConfig) {
     return new Promise((resolve, reject) => {
       pollStatus(guid,
         response => {
-          recordEvent({
-            event: `${formConfig.trackingPrefix}-submission-successful`,
-          });
+          recordEvent({ event: `${formConfig.trackingPrefix}-submission-successful` });
```
[the remark this answers is not in the document]
Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: Good call @erikphansen, yes we should. I will update.

**Review by rianfowler — APPROVED** [review]

File: src/applications/edu-benefits/feedback-tool/helpers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (erikphansen) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: With the new prefix this event string will now have two dashes in a row.

File: src/applications/edu-benefits/feedback-tool/helpers.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ function checkStatus(guid) {
       if (res instanceof Error) {
         Raven.captureException(res);
         Raven.captureMessage('vets_gi_bill_feedbacks_poll_client_error');
+        recordEvent({ event: 'edu-feedback-tool-submission-failed' });
```
Comment (erikphansen) [inline src/applications/edu-benefits/feedback-tool/helpers.js]: Should the prefix now be used to compute these strings?

Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: Good idea, updated.

File: src/applications/edu-benefits/feedback-tool/helpers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: Good catch, removed.

**Review by erikphansen — APPROVED** [review]

**Pull request merged**